ThirdPartyAdapters/unity/unity/src/main/java/com/google/ads/mediation/unity/UnityAdsAdapterUtils.java

```diff
@@ -79,6 +91,19 @@ static AdError createSDKError(@NonNull UnityAds.UnityAdsShowError unityAdsError,
     return new AdError(getMediationErrorCode(unityAdsError), description, SDK_ERROR_DOMAIN);
   }
 
+  /**
+   * Creates a formatted SDK error message based on the specified error code.
+   *
+   * @param errorCode error object from Unity.
+   * @param description   the error message.
+   * @return the error.
+   */
+  @NonNull
+  static AdError createAdError(@NonNull int errorCode,
+                                @NonNull String description) {
```

Comment:
Can be on a single line + we could re-use this method and call from other `createXyzError` since its all common :-)

Reply:
Good idea, did it :)